cyral/resource_cyral_sidecar_listener_test.go

```diff
@@ -355,37 +432,49 @@ func setupSidecarListenerConfig(resourceName string, listener SidecarListener) s
 	}
 
 	var settings string
-	switch {
-	case listener.MySQLSettings != nil:
+
+	if listener.MySQLSettings != nil {
 		dbVersion, charSet := "null", "null"
 		if listener.MySQLSettings.CharacterSet != "" {
 			charSet = fmt.Sprintf(`"%s"`, listener.MySQLSettings.CharacterSet)
 		}
 		if listener.MySQLSettings.DbVersion != "" {
 			dbVersion = fmt.Sprintf(`"%s"`, listener.MySQLSettings.DbVersion)
 		}
-		settings = fmt.Sprintf(
+		settings += fmt.Sprintf(
```

Comment:
`+` doesn't actually look required here, but shouldn't hurt to keep either.

Reply:
This is actually intentional in order to create an invalid configuration (conflicting settings)